addons/wowl/doc/dropdown_component.md

```diff
@@ -0,0 +1,186 @@
+<Dropdown>
+    <!-- "default" slot content should be defined here -->
+    Click me to toggle the dropdown menu !
+    <t t-set-slot="dropdown">
+      <!-- "dropdown" slot content should be defined here-->
+      <DropdownItem>Menu Item 1</DropdownItem>
+      <DropdownItem>Menu Item 2</DropdownItem>
+    </t>
+</Dropdown>
+```
+
+And in the DOM it would get translated similarly to:
+
+```xml
+<div class="o_dropdown">
+  <button>
+    <!-- "default" slot content will take place here -->
+    Click me to toggle the dropdown menu !
+  </button>
+
+  <ul>
+    <!-- "dropdown" slot content will take place here -->
+    <li>Menu Item 1</li>
+    <li>Menu Item 2</li>
+  </ul>
+</div>
+```
+
+### Step 2: make it react to clicks
+
+When a `<DropdownItem/>` get clicked, its parent `<Dropdown/>` will emit a custom `dropdown-item-selected` event containing the payload attached to the clicked item.
+
+If you want to react when a `<DropdownItem/>` get clicked, you need to define two things:
+
+<dl>
+  <dt>The <code>dropdown-item-selected</code> event listener</dt>
+  <dd>It will receive the payload of the item that was clicked.</dd>
+  <dt>A <code>payload</code> for each <code>&lt;DropdownItem/></code> element</dt>
+  <dd>They are just JS objects you declare the way you want. If a payload is not specified, it defaults to <code>null</code>.</dd>
+</dl>
+
+So in your qweb template you would write something like that:
+
+```xml
+<Dropdown t-on-dropdown-item-selected="onItemSelected">
+  …
+  <t t-set-slot="dropdown">
+    …
+    <DropdownItem onClickPayload="{a:15}">Menu Item</DropdownItem>
+    …
+  </t>
+</Dropdown>
+```
+
+In your JS file:
+
+```js
+itemSelected(event) {
+  const eventDetail = event.detail;
+  const itemPayload = eventDetail.payload;
+  console.log(itemPayload.a === 15);
+}
+```
+
+In this case, if you click this menu item, the console will print « true ».
+
+### Step 3: make it shine
+
+Now that you understand the basics of the Dropdown Component, all you need to do is style it the way you want.
+
+✨ Are you ready to make it shine? ✨
+
+If you want to tweak the behavior, check out the available properties in the next section.
+
+## Available Properties
+
+### `<Dropdown/>` props
+
+| Prop name             | Default Value        | Value type | Description                                                    |
+| --------------------- | -------------------- | ---------- | -------------------------------------------------------------- |
+| `open`                | `false`              | boolean    | dropdown starting state - could be used to toggle the dropdown |
+| `hover`               | `false`              | boolean    | open dropdown on hover                                         |
+| `hoverOpenDelay`      | `100`                | number     | delay before dropdown opening when hovered in (in ms)          |
+| `hoverCloseDelay`     | `500`                | number     | delay before dropdown closing when hovered off (in ms)         |
```

Comment:
do we need all this?

Reply:
I am not sure as I do not have the big picture yet.

You must know that for now I dropped the `open` prop because I had an issue with.

I'll try again if we really need it.